src/operation/model_numerical_analysis_utils.jl

```diff
@@ -0,0 +1,130 @@
+# The Numerical stability checks code in this file is based on the code from the SDDP.jl package,
+# from the below mentioned commit and file.
+# commit :8cd305188caffc50a1734913053fc81bba613778
+# link to file :https://github.com/odow/SDDP.jl/blob/d353fe5a2903421e7fed6d609eb9377c35d715a1/src/print.jl#L190
+
+mutable struct NumericalBounds
+    min::Float64
+    max::Float64
+    min_index::Any
+    max_index::Any
+end
+
+NumericalBounds() = NumericalBounds(Inf, -Inf, nothing, nothing)
+
+set_min!(v::NumericalBounds, value::Real) = v.min = value
+set_max!(v::NumericalBounds, value::Real) = v.max = value
+set_min_index!(v::NumericalBounds, idx) = v.min_index = idx
+set_max_index!(v::NumericalBounds, idx) = v.max_index = idx
+
+mutable struct ConstraintBounds
+    coefficient::NumericalBounds
+    rhs::NumericalBounds
+    function ConstraintBounds()
+        return new(NumericalBounds(), NumericalBounds())
+    end
```

Comment:
Making this inner constructor means that you can never instantiate the struct with existing values. Unless that's what you really want, you're probably better off making this an outer constructor.